To: vim_dev@googlegroups.com
Subject: Patch 7.3.071
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.071
Problem:    Editing a file in a window that's in diff mode resets 'diff'
	    but not cursor binding.
Solution:   Reset cursor binding in two more places.
Files:	    src/quickfix.c, src/option.c


*** ../vim-7.3.070/src/quickfix.c	2010-11-24 16:31:55.000000000 +0100
--- src/quickfix.c	2010-12-02 15:02:00.000000000 +0100
***************
*** 2359,2364 ****
--- 2359,2365 ----
  	    set_option_value((char_u *)"bt", 0L, (char_u *)"quickfix",
  								   OPT_LOCAL);
  	    set_option_value((char_u *)"bh", 0L, (char_u *)"wipe", OPT_LOCAL);
+ 	    RESET_BINDING(curwin);
  #ifdef FEAT_DIFF
  	    curwin->w_p_diff = FALSE;
  #endif
*** ../vim-7.3.070/src/option.c	2010-10-13 14:05:29.000000000 +0200
--- src/option.c	2010-12-02 15:12:02.000000000 +0100
***************
*** 9756,9761 ****
--- 9759,9767 ----
  #ifdef FEAT_SCROLLBIND
      to->wo_scb = from->wo_scb;
  #endif
+ #ifdef FEAT_CURSORBIND
+     to->wo_crb = from->wo_crb;
+ #endif
  #ifdef FEAT_SPELL
      to->wo_spell = from->wo_spell;
  #endif
*** ../vim-7.3.070/src/version.c	2010-12-02 14:47:56.000000000 +0100
--- src/version.c	2010-12-02 15:31:12.000000000 +0100
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     71,
  /**/

-- 
A salesperson says:            Translation:
"backward compatible"          Old technology
"Premium"                      Overpriced
"Can't keep it on the shelf"   Unavailable
"Stands alone"                 Piece of shit
"Proprietary"                  Incompatible
				(Scott Adams - The Dilbert principle)

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
